### DISCUSSION TITLE
Touch Screen stop working on convertible devices in tablet mode, uhh!!

### DISCUSSION THREAD

**Discussion by itsManjeet** [itsManjeet/avyos#22] [General]
**How to reproduce**
- Touch screen working fine in normal laptop mode
- Fold device in tablet mode
- After 10 - 20 sec, touch screen stop working?
- Unfold back to laptop mode make things work again

**Comment by ghost** [comment]
*only if you open xwayland application!!

**Reply by itsManjeet** [reply]
yes!!, strange although. I guess need to contact to wayland dev for that if its a known bug
